WalletWasabi/Services/MemPoolBehavior.cs

```diff
@@ -89,7 +89,7 @@ protected override void DetachCore()
 
 		private void ProcessTxPayload(TxPayload transactionPayload)
 		{
-			var transaction = transactionPayload.Object as Transaction;
+			var transaction = transactionPayload.Object;
```

Comment:
Could you make it `Transaction transaction` instead of `var transaction`? For readability.

Reply:
Sure.